.gitignore

```diff
@@ -10,3 +10,7 @@ libnethogs.a
 results/
 build/
 nethogs.egg-info
+
+# IDE configurations
+.vscode
+.idea
```

Comment:
(you can use a [global gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/), but I'm not opposed to adding those)